_tuning-your-cluster/availability-and-recovery/remote-store/migrating-to-remote.md

```diff
@@ -0,0 +1,205 @@
+2. Provide the remote store repository details as node attributes in `opensearch.yml`, as shown in the following example.
+
+   ```yml
+   # Repository name
+   node.attr.remote_store.segment.repository: my-repo-1
+   node.attr.remote_store.translog.repository: my-repo-2
+   
+   # Segment repository settings
+   node.attr.remote_store.repository.my-repo-1.type: s3
+   node.attr.remote_store.repository.my-repo-1.settings.bucket: <Bucket Name 1>
+   node.attr.remote_store.repository.my-repo-1.settings.base_path: <Bucket Base Path 1>
+   node.attr.remote_store.repository.my-repo-1.settings.region: us-east-1
+   
+   # Translog repository settings
+   node.attr.remote_store.repository.my-repo-2.type: s3
+   node.attr.remote_store.repository.my-repo-2.settings.bucket: <Bucket Name 2>
+   node.attr.remote_store.repository.my-repo-2.settings.base_path: <Bucket Base Path 2>
+   node.attr.remote_store.repository.my-repo-2.settings.region: us-east-1
+   
+   ```
+1. Stop the node you are migrating. Do not delete the volume associated with the container when you delete the container. The new OpenSearch container will use the existing volume. **Deleting the volume will result in data loss**.
+1. Deploy a new container running the same version of OpenSearch and mapped to the same volume as the container you deleted.
+1. Query the `_cat/nodes` endpoint after OpenSearch is running on the new node to confirm that it has joined the cluster. Wait for the cluster to become green again.
+1. Repeat steps 2 through 5 for each node in your cluster. 
+1. Reenable shard replication:
+   ```json
+   PUT "/_cluster/settings?pretty"
+   {
+       "persistent": {
+           "cluster.routing.allocation.enable": "all"
+       }
+   }
+   ```
+   The response should look similar to the following example:
+   ```json
+   {
+     "acknowledged" : true,
+     "persistent" : {
+       "cluster" : {
+         "routing" : {
+           "allocation" : {
+             "enable" : "all"
+           }
+         }
+       }
+     },
+     "transient" : { }
+   }
+   ```
+1. Confirm that the cluster is healthy:
+   ```bash
+   GET "/_cluster/health?pretty"
+   ```
+   The response should look similar to the following example:
+   ```json
+   {
+     "cluster_name" : "opensearch-dev-cluster",
+     "status" : "green",
+     "timed_out" : false,
+     "number_of_nodes" : 4,
+     "number_of_data_nodes" : 4,
+     "discovered_master" : true,
+     "active_primary_shards" : 1,
+     "active_shards" : 4,
+     "relocating_shards" : 0,
+     "initializing_shards" : 0,
+     "unassigned_shards" : 0,
+     "delayed_unassigned_shards" : 0,
+     "number_of_pending_tasks" : 0,
+     "number_of_in_flight_fetch" : 0,
+     "task_max_waiting_in_queue_millis" : 0,
+     "active_shards_percent_as_number" : 100.0
+   }
+   ```
+1. The migration to remote store is now complete, and you can begin enjoying the durability and performance benefits.
```

Comment:
We need to revert back to `strict` compatibility mode?

Reply:
done.